### PR TITLE
Stylelint

### PR DESCRIPTION
**Pull request by nickvergessen**
I hope the individual commits make sense

About the config. I now copied the one from photos.
Maybe it makes sense to add a nextcloud one like for the normal linting?

### CODE REVIEW

File: .stylelintrc.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
```
Comment (skjnldsv) [inline .stylelintrc.js:1]: We should have a stylelint nextcloud config, that would be nice xD

Comment (nickvergessen) [reply .stylelintrc.js:1]: Insert "Well now I'm not doing it" meme here

**Comment by juliushaertl** [comment]
Hm not sure what the issue with the stylelint webpack plugin is here:

```
ERROR in No files matching the pattern "/home/jus/repos/nextcloud/server/apps-extra/notestutorial/**/*.s?(c|a)ss" were found.
```

@skjnldsv Any idea about that?

**Comment by skjnldsv** [comment]
Well, there is no files matching the pattern, because they're all vue files?

**Comment by nickvergessen** [comment]
Not sure what this is about, can someone take over?
For me this works in spreed, but maybe there is something hidden in the new config which I'm missing.

**Comment by skjnldsv** [comment]
> For me this works in spr

It works as in "not throw errors"? :p

**Comment by skjnldsv** [comment]
Fixed, anyone have better ideas for the names? 
image

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -29,13 +29,13 @@ jobs:
         run: composer run lint
 
   php-cs-fixer:
-    name: php-cs
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         php-versions: ['7.4']
```
Comment (nickvergessen) [inline .github/workflows/lint.yml:36]: btw php version is in `'` and node version is not. Is there a difference or can we unify?

Comment (skjnldsv) [reply .github/workflows/lint.yml:36]: No idea :woman_shrugging: I guess yaml is smart enough to see it as string?

Comment (juliushaertl) [reply .github/workflows/lint.yml:36]: Well smart is probably not the right word here https://medium.com/@lefloh/lessons-learned-about-yaml-and-norway-13ba26df680 :see_no_evil:

Comment (juliushaertl) [reply .github/workflows/lint.yml:36]: But it should not matter in that case if it is quoted or not.

**Comment by nickvergessen** [comment]
So, good to go?

**Comment by skjnldsv** [comment]
If @juliushaertl is ok :)

**Comment by juliushaertl** [comment]
Yep, all good from my side.

**Pull request merged**